### DISCUSSION TITLE
ананимно песать указать некнейм

### DISCUSSION THREAD

**Discussion by ExtinctDrift19** [OpenVK/openvk#296] [Ideas]
Как на двач добавить в openvk имеджборд создать

**Comment by ghost** [comment]
... а теперь по-русски.

**Comment by NoPlagiarism** [comment]
Что OpenVk, что ВК, не созданы как имиджборд. Но анонимно написать возможность есть, бех всяких никнеймов

**Comment by NoPlagiarism** [comment]
(пожалуйста, оставьте этот вопрос отвеченным. Их стало слишком много)

**Comment by NoPlagiarism** [comment]
@dsrev а зачем анлокнул вообще?
